SpatialOS/scripts/build-project.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -e -x
+
+cd "$(dirname "$0")"
+
+echo $(pwd)
+
+WORKER_DIRS=(HelloWorker DiceWorker client)
+SDK_VERSION="13.5.1"
+BUILD_DIR="$(pwd)/../.."
```

Comment:
This concept of a root dir doesn't seem to be a useful concept for FPL where we allow workers to be freely put everywhere. How about hardcoding the dir for each worker instead of having a "hack" that assumes that all workers are inside the same parent folder?

Reply:
done